packages/@aws-cdk/aws-rds/package.json

```diff
@@ -112,6 +112,7 @@
       "props-physical-name:@aws-cdk/aws-rds.DatabaseInstanceReadReplicaProps",
       "props-physical-name:@aws-cdk/aws-rds.DatabaseSecretProps",
       "props-physical-name:@aws-cdk/aws-rds.OptionGroupProps",
+      "props-physical-name:@aws-cdk/aws-rds.SubnetGroupProps",
```

Comment:
I see `SubnetGroupProps` have a `subnetGroupName` property... is that not the physical name?

Reply:
The current `props-physical-name` linter rule is pretty strict. The name _must_ be prefixed with the Cfn name, which in this case is `DBSubnetGroup`. So the physical name would need to be `dbSubnetGroupName` for the linter rule to match; I just couldn't bring myself to do that.

I would love to make the linter more flexible so we can otherwise indicate an attribute as the physical name, but I figured that was out of scope.

Reply:
Ah, I forgot the class was named `DBSubnetGroup`. Yep, makes sense, I would leave it as `subnetGroupName` as well in that case.